software_reseller/data/ir_ui_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="project_project_form_customization" model="ir.ui.view">
+        <field name="name">project.project.form customization</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.edit_project"/>
+        <field name="mode">extension</field>
+        <field name="priority">240</field>
+        <field name="type">form</field>
+        <field name="active" eval="True"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='settings']/group[2]" position="inside">
```

Comment:
More stable?
```suggestion
            <xpath expr="//page[@name='settings']//group[@name='group_documents_analytics']" position="inside">
```

Reply:
It is not working 
![image](https://github.com/user-attachments/assets/7bfed817-4395-49b3-be33-0c49e66994d8)